assets/RNAseq_report.Rmd

```diff
@@ -875,57 +925,145 @@ The read sequencing quality was analyzed using `FASTQC`. More detailed informati
 
 The read mapping and gene assignment statistics, together with other quality control parameters are summarized here.
 
-The table below, extracted from the `MultiQC` report, shows a summary of the bioinformatics analysis quality control.
-Note: Duplicate intercept (%) is the percentage of duplicate reads for the intercept of a linear model of duplicate reads vs read counts. High numbers of duplicates at low read counts can indicate low library complexity with technical duplication."
+The table below, extracted from the `MultiQC` report, shows a summary of the bioinformatics analysis quality control.",
+ifelse(mqc_version == 'old_mqc',
+"Note: Duplicate intercept (%) is the percentage of duplicate reads for the intercept of a linear model of duplicate reads vs read counts. High numbers of duplicates at low read counts can indicate low library complexity with technical duplication.",
+"")
```

Comment:
Is that empty string there for a reason?

Reply:
It's there to make the ifelse work 😅 Not sure if there is a more efficient way of constructing these pasted strings